### PR TITLE
A couple of fixes for chrome.storage migration

### PR DESCRIPTION
**Pull request by kav2k**
(fix for #16)

### CODE REVIEW

File: Chrome/localStorage-proxy.js [hunk 1]
```diff
@@ -20,6 +20,7 @@
         // Migrate from previous version of the add-on, that used the real
         // localStorage object to persist preferences.
         var localStorageKeyCount = localStorage.length;
+        storage = storage || {};
```
Comment (Rob--W) [inline Chrome/localStorage-proxy.js]: Please change it to `storage = {};`

**Comment by Rob--W** [comment]
And squash the commits please, thanks! Feel free to ask if you don't know how to squash commits.

**Comment by kav2k** [comment]
Done.

**Comment by Rob--W** [comment]
Its only bad form to amend the commit if you know that the branch is also used by others. Since the development branch in your repository is completely yours, it's no problem at all to squash the commits. Thanks for the patch!

**Pull request merged**